### DISCUSSION TITLE
feature requests for computational notebooks

### DISCUSSION THREAD

**Discussion by tonyfast** [deathbeds/nbconvert-a11y#45] [General]
a place to  collect what the notebooks for all community desires for assistive computations documents

**Comment by ecmandell** [comment]
Hey Tony, is it possible to get a list of current features and a brief description of each, so I don't suggest features that already exist.

**Reply by tonyfast** [reply]
i don't know how soon i can produce a feature list. some of the features are outcomes of chooses i made; i may not recognize them as features. i'd appreciate it if you could share your candid opinion, to the extent you feel comfortable. 

the best place to find a list of these features is a the class definition of the `nbconvert-a11y` exporter. all the of the traits defined the [`A11yExporter` class](https://github.com/deathbeds/nbconvert-a11y/blob/main/nbconvert_a11y/exporter.py#L97) are features that i've considered. they exist to varying degrees of documentation.

**Comment by ecmandell** [comment]
Thanks Tony, I'll check it out. I'll start poking around and trying things out and keep notes as I go.
